apps/tx-builder/src/components/Builder.tsx

```diff
@@ -266,15 +263,12 @@ export const Builder = ({
     onRemoveTransaction(inputIndex);
   };
 
-  const isValidAddress = useCallback(
-    (address: string | null) => {
-      if (!address) {
-        return false;
-      }
-      return services?.web3?.utils.isAddress(address);
-    },
-    [services.web3],
-  );
+  const isValidAddress = useCallback((address: string | null) => {
```

Comment:
this can be moved outside the component now because it doesn't depend on anything inside the component now

Reply:
nice catch! done! :D